OWNERS

```diff
@@ -0,0 +1,6 @@
+# approval == this is a good idea /approve
+approvers:
+  - nbalach
+# review == this code is good /lgtm
+reviewers:
+  - nbalacha
```

Comment:
It will be good to add more people to that list. Such as @leelavg @rohantmp @sp98.

Reply:
They will be added later. This is an initial file PR.